test/scale-infra/README.md

```diff
@@ -0,0 +1,227 @@
+    ```
+
+    The above command creates the required AWS infrastructure including a VPC, security groups and EC2 instances. Then, it creates a Kubernetes cluster deployed using Kubeadm. Lastly, it deploys the KubeStellar core components. You can use the flag `--ks-release` to specify the KubeStellar release. Kubestellar is deployed using the [KS helmchart](https://github.com/kubestellar/kubestellar/tree/release-0.26.0/core-chart) configured with a ITS of type host. 
+
+    Use the flag `--vpc-name` to specify the name for the [AWS virtual private cloud](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html) to deploy your infrastructure in a logically isolated virtual network: *We highly advise utilizing a unique name or the AWS IAM user ID as the identifier for your VPC*. Furthermore, use the flag `--ec2-image-id` to specify the [Amazon machine image ID](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html), keeping in mind that it is region-specific.
+
+    We advise to use the Ubuntu Amazon EC2 AMI Locator to acquire the AMI value for a specific region: https://cloud-images.ubuntu.com/locator/ec2/. Select Ubuntu release `22.04` or `24.04`. 
+
+    Also, use the flag `--aws-key-name` to specify the name of the uploaded public ssh key to your AWS account in the target region. 
+
+
+    Extract the command displayed at the end of the last Ansible task to edit your `etc/hosts` file with wds1 host entry:
+
+    (*Sample Ansible task command line output*):
+
+    ```console
+    TASK [Command to edit your etc/hosts file with wds1 host entry] ***********************************************************************
+    ok: [master1] => {
+        "msg": "echo '192.168.56.20 wds1.ec2-192-168-56-20.compute-1.amazonaws.com' | sudo tee -a /etc/hosts"
+    }
+
+    PLAY RECAP ****************************************************************************************************************************
+    master1                    : ok=24   changed=20   unreachable=0    failed=0    skipped=2    rescued=0    ignored=1   
+    ```
+
+    Then, extract and execute the command displayed in the Ansible task above:
+
+    ```bash
+    echo '192.168.56.20 wds1.ec2-192-168-56-20.compute-1.amazonaws.com' | sudo tee -a /etc/
+    ```
+    
+    Upon completion of the script's execution, an Ansible inventory file containing the IP addresses of the nodes that constitute the Kubernetes cluster will be generated at the current directory at `.data/hosts_core`.
+
+    Furthermore, use the generated kubeconfig file `.data/admin.conf` to access the Kubernetes cluster and the control plane components for KubeStellar, for example:
+    
+    Check the created kubeconfig contexts:
+
+    ```bash 
+    kubectl --kubeconfig=${KUBECONFIG:-$PWD/admin.conf} config get-contexts
```

Comment:
If the user is specifically supposed to use `$PWD/admin.conf` then why is `$KUBECONFIG` in here? Same comment applies in all the copies of this pattern.

Reply:
Fixed!